.github/ISSUE_TEMPLATE/request-for-position.yml

```diff
@@ -0,0 +1,84 @@
+name: Request for WebKit's position on an emerging web specification
+description: If you're working on a spec and would like to know the WebKit Open Source Projet's position on it, this is the form for you!
+body:
+  - type: markdown
+    attributes:
+      value: "## WebKittens who can provide input"
+  - type: input
+    id: webkittens
+    attributes:
+      label: "WebKittens"
+      description: "Who from the WebKit community do you think is best positioned to review this spec? It's okay if you don't know."
+      placeholder: "@annevk @hober @marcoscaceres"
+  - type: markdown
+    attributes:
+      value: "## Information about the specification or proposal"
+  - type: input
+    id: spec-title
+    attributes:
+      label: "Title of the spec"
+      placeholder: "CSS Frobnication Module, Level 5"
+    validations:
+      required: true
+  - type: input
+    id: spec-link
+    attributes:
+      label: "URL to the spec"
+      placeholder: "https://www.w3.org/TR/SHORTNAME"
+    validations:
+      required: true
+  - type: input
+    id: spec-repo
+    attributes:
+      label: "URL to the spec's repository"
+      description: "On GitHub or whatever repository hosting provider the spec uses."
+      placeholder: "https://github.com/ORG/REPO"
+  - type: input
+    id: spec-issues
+    attributes:
+      label: "Issue Tracker URL"
+      description: "Only supply this if the spec doesn't use GitHub for issue tracking"
+  - type: input
+    id: spec-explainer
+    attributes:
+      label: "Explainer URL"
+      description: "Only supply this if the explainer isn't the README.md in the GitHub repo"
+  - type: markdown
+    attributes:
+      value: "## Design reviews and vendor positions"
+  - type: input
+    id: tag-review
+    attributes:
+      label: "TAG Design Review URL"
+      placeholder: "https://github.com/w3ctag/design-reviews/issues/NNN"
+  - type: input
+    id: moz-position
+    attributes:
+      label: "Mozilla standards-positions issue URL"
+      placeholder: "https://github.com/mozilla/standards-positions/issues/NNN"
+  - type: markdown
+    attributes:
+      value: "## Bugs tracking this feature"
+  - type: input
+    id: webkit-bugzilla
+    attributes:
+      label: "WebKit Bugzilla URL"
+      placeholder: "https://webkit.org/b/NNNNNN"
+  - type: input
+    id: radar
+    attributes:
+      label: "Radar URL"
```

Comment:
a lot of people won't know what this is and I wonder if we should include it at all. If there is a bugzilla bug already, there will likely be a radar linked from it.... maybe we should drop this?

Reply:
It hasn't been an issue with the current template I think and it's kinda useful to have it. I suggest we keep it for now.

Reply:
imo, I think it's not a bad thing to discuss (again), but I don't think we should block changing the format of the form on this